### DISCUSSION TITLE
Replicating zk's fzf.vim picker, but with fzf-lua instead?

### DISCUSSION THREAD

**Discussion by megalithic** [zk-org/zk-nvim#109] [Help] [answered]
I'm presently working with `fzf-lua` instead of `fzf/fzf.vim` and would love to replicate the basic functionality around zk's built in fzf picker behaviours and use fzf-lua instead. Was wondering if any zk users would chime in if they've done something similar.

**Comment by vicrdguez** [comment, marked as answer]
Hi @megalithic I'm also using `fzf-lua` so I implemented the picker myself. The PR is open: #138 

If the maintainers are happy with it, hopefully we can get it merged, it should have full support for all the functionality present for other pickers. I've daily driving `fzf-lua` with `zk-nvim` and works wonders
